qipr_approver/approver/tests/tests_models/test_approve_models.py

```diff
@@ -1,38 +1,108 @@
 import sys
 
 from approver.models import *
+from approver.tests.tests_models.test_utils import check_fields, is_foreign_key_to
 
 import django
 from django.contrib.auth.models import User
 from django.core.urlresolvers import reverse
 from django.db.models import fields, signals
 from django.test import TestCase, Client
+from django.db.utils import IntegrityError
 
 class SectionTestCase(TestCase):
     def setUp(self):
-        pass
+        self.user = User(username = 'testuser')
+        self.user.save()
 
-    def test_no_section_tests_written(self):
-        self.fail('You have not written any tests for {}'.format(type(self)))
+    def test_should_pass_when_model_has_correct_fields(self):
+        self.assertTrue(check_fields(Section, "name", "Char"))
+        self.assertTrue(check_fields(Section, "sort_order", "Integer"))
+
+    def test_should_throw_exception_when_not_unique_sort_order(self):
+        section1 = Section(name = 'section1', sort_order = 1)
+        section2 = Section(name = 'section2', sort_order = 1)
+
+        section1.save(self.user)
+        self.assertRaises(IntegrityError, lambda : section2.save(self.user))
+
+    def test_should_print_name_when_asked_for_string(self):
+        text = "sample text"
+        section = Section(name = text)
```

Comment:
bad style to put spaces around equals signs when invoking a function with keyword args [https://www.python.org/dev/peps/pep-0008/#other-recommendations](referenced here)